runtime/src/mips.rs

```diff
@@ -299,9 +313,37 @@ decl_module! {
             Self::close_proposal(proposal_hash.clone());
         }
 
+        /// An emergency proposal that bypasses network voting process. Governance committee can make
+        /// a proposal that automatically becomes a referendum on which the committee can vote on.
+        #[weight = SimpleDispatchInfo::FixedOperational(200_000)]
+        pub fn emergency_referendum(origin, proposal: Box<T::Proposal>) {
+            // Proposal must originate from the committee
+            T::CommitteeOrigin::try_origin(origin)
+                .map(|_| ())
+                .or_else(ensure_root)
+                .map_err(|_| "bad origin")?;
+
+            let proposal_hash = T::Hashing::hash_of(&proposal);
+
+            // Proposal must be new
+            ensure!(!<Proposals<T>>::exists(proposal_hash), "proposal from committee already exists");
+
+            let index = Self::proposal_count();
+            <ProposalCount>::mutate(|i| *i += 1);
+
+            <ReferendumMetadata<T>>::mutate(|metadata| metadata.push((index, MipsPriority::High, proposal_hash)));
```

Comment:
Would it make sense to reuse `create_referendum` (and add a param to that function for priority)?

Reply:
One would assume so! Not sure why I didn't think of that. I've used `create_referendum` now.